### PR TITLE
Devdocs 2655 - add information about $refinery filter

### PR DESCRIPTION
**Pull request by markcmurphy**
# [DEVDOCS-2655](https://jira.bigcommerce.com/browse/DEVDOCS-2655)

## What changed?
* added usage notes section to be expanded upon
* added Classes -> Refinery usage clarification

### CODE REVIEW

File: docs/bigcommerce-for-wordpress/extending-the-plugin/code-reference.md [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
-
 # Code Reference
 
 <div class="otp" id="no-index">
 
-### On This Page
-- [Code Reference](#code-reference)
+### On this page
+- [Code reference](#code-reference)
+- [Useage notes](#useage-notes)
```
Comment (ektravel) [inline docs/bigcommerce-for-wordpress/extending-the-plugin/code-reference.md:7]:
```suggestion
- [Usage notes](#usage-notes)

```

File: docs/bigcommerce-for-wordpress/extending-the-plugin/code-reference.md [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
 
 </div> 
 
-## Code Reference
-Search the BigCommerce for WordPress codebase for detailed entries on available hooks, functions, classes and methods. Do a deep dive, and browse the building blocks you can use to extend and customize the plugin.
+## Code reference
+To search the BigCommerce for WordPress (BC4WP) codebase for detailed entries on available hooks, functions, classes, and methods, visit the [BigCommerce Plugin Code Reference](https://bigcommerce.moderntribe.qa/).
+
+
+## Useage notes
```
Comment (ektravel) [inline docs/bigcommerce-for-wordpress/extending-the-plugin/code-reference.md:15]:
```suggestion
## Usage notes

```

File: docs/bigcommerce-for-wordpress/extending-the-plugin/code-reference.md [hunk 3]
```diff
@@ -1,15 +1,21 @@
...
+
+The following list contains additional notes on the usage of the hooks, functions, classes, and methods found in the [BigCommerce Plugin Code Reference](https://bigcommerce.moderntribe.qa/) that allow you to extend and customize the BC4WP plugin:
+
+### Classes
 
-[View the code reference here](https://bigcommerce.moderntribe.qa/)
+  - Refinery - refinery only filters product results on product archive or product category archive pages.
```
Comment (ektravel) [inline docs/bigcommerce-for-wordpress/extending-the-plugin/code-reference.md:21]:
```suggestion
  - Refinery - filters product results on product archive or product category archive pages.

```

**Review by ektravel — COMMENTED** [review]
Added a few suggestions.

**Comment by markcmurphy** [comment]
> Added a few suggestions.

@ektravel Very much appreciated! I just merged those suggestions.

**Review by ektravel — APPROVED** [review]
LGTM

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**